conference/CG/sca.yml

```diff
@@ -0,0 +1,15 @@
+- title: SCA
+  description: ACM SIGGRAPH/Eurographics Symposium on Computer Animation
+  sub: CG
+  rank: B
+  dblp: sca
+  confs:
+    - year: 2024
+      id: sca24
+      link: https://computeranimation.org/
+      timeline:
+        - deadline: '2024-04-22 23:59:59'
+          comment: 'Submission deadline (papers and posters)'
+      timezone: UTC-0
+      date: August 21-23, 2022
```

Comment:
should be 2024?

Reply:
Also please check the timezone selection~ 
<img width="207" alt="image" src="https://github.com/ccfddl/ccf-deadlines/assets/20631913/b5a84b5b-b6c8-472c-8ad4-1f50f2561907">